crud/src/main/java/com/redhat/lightblue/rest/crud/health/CrudCheckResponse.java

```diff
@@ -24,6 +24,13 @@ public JsonNode toJson() {
             ObjectNode resultResponse = factory.objectNode();
             resultResponse.put("healthy", result.getValue().isHealthy());
             resultResponse.put("message", result.getValue().getMessage());
+
+            if (result.getValue().getDetails() != null) {
+              for (Map.Entry<String, Object> entry : result.getValue().getDetails().entrySet()) {
+                resultResponse.put(entry.getKey().toString(), entry.getValue().toString());
```

Comment:
nit: redundant toString on the key

Also, IMO let's add "timestamp" as a field, too, from getValue().getTimestamp()